hathor/indexes/manager.py

```diff
@@ -176,8 +282,8 @@ def enable_tokens_index(self) -> None:
 class RocksDBIndexesManager(IndexesManager):
     def __init__(self, db: 'rocksdb.DB') -> None:
         from hathor.indexes.memory_deps_index import MemoryDepsIndex
+        from hathor.indexes.memory_mempool_tips_index import MemoryMempoolTipsIndex
```

Comment:
Why are we using Memory for mempool instead of RocksDB?

Reply:
For performance reasons, I added a comment bellow.

Reply:
Thanks!